lms/djangoapps/discussion/rest_api/forms.py

```diff
@@ -181,8 +188,12 @@ def clean_rolename(self):
         if course_id and rolename:
             try:
                 role = Role.objects.get(name=rolename, course_id=course_id)
-            except Role.DoesNotExist:
-                raise ValidationError(f"Role '{rolename}' does not exist")  # lint-amnesty, pylint: disable=raise-missing-from
+            except Role.DoesNotExist as err:
+                raise ValidationError(f"Role '{rolename}' does not exist") from err
 
             self.cleaned_data['role'] = role
             return rolename
+
+
+class UserCourseStatsForm(_PaginationForm):
+    order_by = ChoiceField(choices=UserOrdering.choices, required=False)
```

Comment:
We should set `default` value here to `UserOrdering.BY_ACTIVITY` - this can simplify code later when we are checking the value of `order_by`

Reply:
The default depends on the type of user so setting it here will make it harder to know if the value comes from the default or from the user.